garrysmod/lua/vgui/dcolormixer.lua

```diff
@@ -307,7 +307,7 @@ function PANEL:UpdateConVar( strName, strKey, color )
 	if ( !strName ) then return end
 	local col = color[strKey]
 
-	RunConsoleCommand( strName, tostring( col ) )
+	RunConsoleCommand( strName, col:ToString() )

```

Comment:
If `COLOR:ToString` changes output - this will break everything that uses DColorMixers.